cli/internal/cmd/configfetchmeasurements.go

```diff
@@ -95,47 +111,54 @@ func configFetchMeasurements(
 		return err
 	}
 
+	cfm.log.Debugf("Fetched and verified measurements, hash is %s", hash)
 	if err := verifyWithRekor(cmd.Context(), verifier, hash); err != nil {
 		cmd.PrintErrf("Ignoring Rekor related error: %v\n", err)
 		cmd.PrintErrln("Make sure the downloaded measurements are trustworthy!")
 	}
 
+	cfm.log.Debugf("Verified measurements with Rekor, updating measurements in config")
 	conf.UpdateMeasurements(fetchedMeasurements)
 	if err := fileHandler.WriteYAML(flags.configPath, conf, file.OptOverwrite); err != nil {
 		return err
 	}
-
+	cfm.log.Debugf("Wrote configuration to YAML")
 	return nil
 }
 
 // parseURLFlag checks that flag can be parsed as URL.
 // If no value was provided for flag, nil is returned.
-func parseURLFlag(cmd *cobra.Command, flag string) (*url.URL, error) {
+func (cfm *configFetchMeasurementsCmd) parseURLFlag(cmd *cobra.Command, flag string) (*url.URL, error) {
 	rawURL, err := cmd.Flags().GetString(flag)
 	if err != nil {
 		return nil, fmt.Errorf("parsing config generate flags '%s': %w", flag, err)
 	}
+	cfm.log.Debugf("Flag %s has raw URL %s", flag, rawURL)
 	if rawURL != "" {
+		cfm.log.Debugf("Parsing raw URL %s", rawURL)
 		return url.Parse(rawURL)
 	}
 	return nil, nil
 }
 
-func parseFetchMeasurementsFlags(cmd *cobra.Command) (*fetchMeasurementsFlags, error) {
-	measurementsURL, err := parseURLFlag(cmd, "url")
+func (cfm *configFetchMeasurementsCmd) parseFetchMeasurementsFlags(cmd *cobra.Command) (*fetchMeasurementsFlags, error) {
+	measurementsURL, err := cfm.parseURLFlag(cmd, "url")
 	if err != nil {
 		return &fetchMeasurementsFlags{}, err
 	}
-
-	measurementsSignatureURL, err := parseURLFlag(cmd, "signature-url")
+	cfm.log.Debugf("Parsed measurements URL")
+	
```

Comment:
There is a tab in this line that gofmt doesn't like, the line should just be empty.